src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -255,8 +270,8 @@ def get(self, request: Request, organization: Organization | RpcOrganization) ->
                     enriched_threshold.update(
                         {
                             "key": self.construct_threshold_key(release=release, project=project),
-                            "start": release.date,  # deploy.date_finished _would_ be more accurate, but is not keyed on project so cannot be used
-                            "end": release.date
+                            "start": threshold_start,
+                            "end": threshold_start
```

Comment:
these are always the same?

Reply:
Yes - this is simply a starting point - but as we iterate through thresholds, we modify this value so that the `start` is the earliest value, and the `end` is the latest value

Reply:
Oh oops - actually, here the `end` is the `threshold_start + timedelta`